readme.md

```diff
@@ -37,6 +37,22 @@ python manage.py runserver
 
 If there are new changes to the database the runserver output will run you through the process of updating and running the migrations.
 
+#### Running RabbitMQ & Celery independently
```

Comment:
This is currently a subsection of the `Ubuntu based Linux` section. Is that meant to be the case?

Reply:
I don't have any instructions for doing it in Windows, so I was not sure that it would be the same there too

Reply:
I have made a commit addressing this as of 2 mins ago